### DISCUSSION TITLE
Welcome to ATAC Discussions!

### DISCUSSION THREAD

**Discussion by Julien-cpsn** [Julien-cpsn/ATAC#52] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by antoniorodr** [comment]
It would be nice to be able to create documentation from ATAC. Maybe the program can write it on a .md file or something, from a default template, importing the request and everything you have for the project.

**Reply by Julien-cpsn** [reply]
Very great idea. Added in README goals.
